duckduckgo/macos-browser
# fix popover message bug

Task/Issue URL:  https://app.asana.com/0/392891325557410/1200812097536610
Tech Design URL:
CC:

**Description**:

Fix bug where message won't show when other buttons are visible.

Please merge on approve.

**Steps to test this PR**:
1. Open the downloads dialog so the downloads button appears
1. Create a private email address
2. Ensure popover message is visible

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
